pkg/api/meta/restmapper.go

```diff
@@ -335,3 +359,23 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
 	}
 	return nil, false
 }
+
+// ResourceInGroup verifies if the group has the resource in it
+func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
+	for _, t := range m {
+		if t.ResourceInGroup(group, resource) {
+			return true
+		}
+	}
+	return false
+}
+
+// IsResource takes a string and checks if it's a resource
```

Comment:
I feel like this is missing the concept of ambiguous resources: what if `experimental/daemonset` and `controllers/daemonset` both exist, and resource is just `daemonset`?

Also comment on how fully qualified `resource` is expected to be.